test/dispatch-ci.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This script is exercised from Dispatch.
+
+set -euxo pipefail
+
+# Tests are exercised from `test` directory. The current directory of the script.
+cd "$(dirname "$0")"
+
+echo "Setup Konvoy"
+source ./scripts/setup-konvoy.sh v1.4.1
+
+echo "git fetch branches."
+git fetch
+
+echo "INFO: Exercise TestAwsGroup "
+go test -tags experimental -timeout 60m -race -v -run TestAwsGroup
+
+# Fix kind cluster issues on Dispatch CI before enabling complete test-suite.
```

Comment:
Thanks for the comments. Just a note, doing this will reduce the tests actually done [on dispatch] down to almost nothing except for AWS storage addons.

Reply:
Got it. I hope we can kind tests effectively on dispatch and TODO is resolved. Until then, this has some value.

Thank you!